source/_integrations/opensky.markdown

```diff
@@ -18,6 +18,23 @@ The OpenSky integration allows one to track overhead flights in a given region.
 
 {% include integrations/config_flow.md %}
 
+{% configuration_basic %}
+latitude:
+  description: The latitude of the center of the area to track.
+longitude:
+  description: The latitude of the center of the area to track.
+radius:
+  description: The radius in meters around the latitude/longitude point to track.
+altitude:
+  description: The minimum height in meters above which the planes are tracked. No input (or 0) means all planes are tracked in the area.
+{% endconfiguration_basic %}
+
+## Authentication
+
+By default, to not hit the API limit, polling is done every 15 minutes.
+You can do more API requests as an authenticated user.
+To login you can enter your credentials in the integration configuration.
```

Comment:
I would rewrite this sentence to inform the user that the credentials must be inserted via the options flow.
Otherwise, they need help figuring out where to insert the credentials during the config entry setup.

Reply:
Yes, but we know what an option flow is, they only know what configuring an integration is, so have any suggestions on wording?

Reply:
@edenhaus gave another proposal:
"More api requests are allowed as authenticated user. You can add your credentials after setting up the config entry"
But I wasn't sure config entry is a used word in the documentation. Do you have an idea @c0ffeeca7?

Reply:
Yes, config entry is already used in 91 files.